doc/ResultsDiscussionWeaning.Rmd

```diff
@@ -0,0 +1,40 @@
+```{r}
+library(tidyverse)
+library(ggplot2)
+library(lme4)
+library(broom)
+library(ggthemes)
+library(MuMIn)
+
+#devtools::load_all(".")
+
+
+fit <- lmer(log.oy ~ log.wm + (log.wm | order), data = mammals_sub)
+summary(fit)
+lme4::ranef(fit)
+tidy(fit, conf.int = TRUE)
+broom::augment(fit) %>%
+    ggplot(aes(x = log.wm, y = .fixed)) +
+    geom_line() +
+    geom_point(aes(x = log.wm, y = log.oy, color = order), alpha = 0.15) +
+    geom_line(aes(y = .fitted, color = order)) #+
+    #team_theme() + ylab("Number of offspring per year") + xlab("Weaning age (months)")
+
+```
+
+
+
+##Discussion
+###Weaning age
+One of the biggest investments that a mother makes towards her offspring is lactation. In fact, lactation has been described as the most costly resource allocation (Gittleman and Thomspon 1988).As predicted by parent-offspring conflict (Trivers 1974), the ideal optima in reproductive decisions differ in parents and offspring. Since lactation is an important factor that requires resources from mother, it is crucial for the parents to determine for how long to nurse their offspring. Parents� strategy aims at maximizing (future) reproduction without foregoing the needs of the current litter. Stearns (1989) described this as tradeoff between somatic and reproductive investments. From offspring�s perspective, prolonging the nursing/lactation period for as long as possible is the best strategy because transition from lactating to weaning imposes stress on the offsring. For example, separation from mother has been associated with increased amounts of cortisol in otters (Malmkvist et al. 2016). 
+In our data analysis, we found a negative correlation between the number of offspring a female has in one year, and the age at which her offspring are weaned, as is expected by the trade-off hypothesis (Trivers 1974). Moreover, out data shows clear distinction between mammals with high and low reproductive output (Fig. S4) Rodents are expected to have resources for more offspring, given that their investment in each litter is low. Previous research has found that increased period of lactation delays the time of birth of the next litter (Hedrich and Bullock 2004). This is an indication of the conflict of future and current offspring. As a result of that, and the fact that mice have relatively short life span (due to high metabolic rate (Hedrich and Bullock 2004)), mothers cannot allocate as many resources towards any given offspring, and therefore most rodents start weaning at approximately 2 months of age (Figure S4). Mice also show an ability to adjust both the quality and quantity of produced milk according to litter size (Hedrich and Bullock 2004). Despite this, offspring reared in high litter batches usually have lower weight. This is consistent with what we observe in our data, and suggests that energy expenditure for lactation determines the amount of offspring the rodent produces. 
```

Comment:
You can add an indication/measurement of how strong the correlation is in a parenthesis after "... found a negative correlation"